src/main/java/org/sagebionetworks/web/client/widget/entity/EntityListRowBadge.java

```diff
@@ -183,21 +191,36 @@ public void onSelectionChanged() {
 
 	@Override
 	public void onAddToDownloadList() {
```

Comment:
Adding a single file to the DL from an Entity List (each row is an EntityListRowBadge).  Use new service if in alpha mode.